# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mediacontrol/tests/gtest/TestMediaKeysEvent.cpp
# Commit: 714b2fb21e23
# Full Hash: 714b2fb21e2341e8e2bf1be1271ca0ed5e526443
# Author: alwu <alwu@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1640998
# File Overlap Count: 1
# Description:
#   Bug 1640998 - part5 : set supported keys to the event source r=chunmin
#   
#   This patch will
#   - add a method `SetSupportedMediaKeys()` on `MediaControlKeysEventSource`
#   - set main controller's supported key to the event source
# ==============================================================================

diff -r a48b6554ba9c -r 714b2fb21e23 dom/media/mediacontrol/tests/gtest/TestMediaKeysEvent.cpp
--- a/dom/media/mediacontrol/tests/gtest/TestMediaKeysEvent.cpp	Mon Jun 08 22:16:46 2020 +0000
+++ b/dom/media/mediacontrol/tests/gtest/TestMediaKeysEvent.cpp	Tue Jun 09 02:23:35 2020 +0000
@@ -13,6 +13,7 @@
   NS_INLINE_DECL_REFCOUNTING(MediaControlKeysEventSourceTestImpl, override)
   bool Open() override { return true; }
   bool IsOpened() const override { return true; }
+  void SetSupportedMediaKeys(const MediaKeysArray& aSupportedKeys) override {}
 
  private:
   ~MediaControlKeysEventSourceTestImpl() = default;